staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/start.go

```diff
@@ -78,7 +79,11 @@ func NewCommandStartCustomResourceDefinitionsServer(out, errOut io.Writer, stopC
 }
 
 func (o CustomResourceDefinitionsServerOptions) Validate(args []string) error {
-	return nil
+	errors := []error{}
+	if errs := o.RecommendedOptions.Validate(); len(errs) != 0 {
```

Comment:
actually. We not: `errors = append(errors, o.RecommendedOptions.Validate()...)`? We don't do that anywhere. But it would be much shorter and equivalent.